cognite/powerops/utils/mapping/mapping.py

```diff
@@ -0,0 +1,41 @@
+from cognite.client import CogniteClient
+
+from cognite.powerops.data_classes.time_series_mapping import TimeSeriesMapping
+from cognite.powerops.utils.common import print_warning
+
+
+def filter_time_series_mappings(mapping: TimeSeriesMapping, client: CogniteClient) -> TimeSeriesMapping:
+    """Filters out mapping entries where the expected CDF time series is not found."""
+    if not mapping.rows:
+        print_warning("TimeSeriesMapping is empty! No filtering possible!")
+        return mapping
+
+    external_ids_in_mapping = {
+        mapping.time_series_external_id for mapping in mapping if mapping.time_series_external_id is not None
+    }
+    time_series_in_cdf = client.time_series.retrieve_multiple(
+        external_ids=list(external_ids_in_mapping), ignore_unknown_ids=True
+    )
+    external_ids_in_cdf = {time_series.external_id for time_series in time_series_in_cdf}
```

Comment:
```suggestion
    external_ids_in_cdf = {
        time_series.external_id
        for time_series in client.time_series.retrieve_multiple(
            external_ids=list(external_ids_in_mapping), ignore_unknown_ids=True
        )
    }
```

just a suggestion, realize the `time_series_in_cdf` is not use for anything else than the e_ids